packages/stegoveritas/PKGBUILD

```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=stegoveritas
```

Comment:
You can add here a `_pkgname=stegoVeritas` in order to use `$_pkgname-$pkgver` when doing the `cd` in build() and package().